hazelcast/src/test/java/com/hazelcast/test/annotation/LiveTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+
+/**
+ * Annotates tests which requires integration tests over a live api.
+ * <p/>
+ * Will be executed in nightly builds and for jclouds live tests
+ */
+public final class LiveTest {
```

Comment:
Sure, Simulator also uses jclouds in the Provisioner tool to support all kind of cloud services. But as I said, during the nightly tests we had issues with it (and sometimes even with the AWS CLI utils, I suspect peaks at AWS causing issues sometimes).

I think we can run those tests regularly, but maybe not nightly (the API will surely not change that often). For a nightly test the mocking approach maybe more suitable.

This is my abstract test class where I have an init method for the `ComputeService` mock:
https://github.com/hazelcast/hazelcast-simulator/blob/master/simulator/src/test/java/com/hazelcast/simulator/provisioner/AbstractComputeServiceTest.java

This is an example usage where I use the `initComputeServiceMock()` in the `@Before` method and some `verify()` checks in the `@Test` methods:
https://github.com/hazelcast/hazelcast-simulator/blob/master/simulator/src/test/java/com/hazelcast/simulator/provisioner/TemplateBuilderTest.java